app/customer.py

```diff
@@ -0,0 +1,59 @@
+# from datetime import datetime
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self, info: dict) -> None:
+        self.name = info["name"]
+        self.location = info["location"]
+        self.money = info["money"]
+        self.car_consumption = info["car"]["fuel_consumption"]

```

Comment:
why `car_fuel_consumption` is `Customer`'s instance attribute?
Maybe it is better to implement a separate `Car` class, what do you think?